app/routers/users.py

```
@@ -274,4 +279,28 @@ async def delete_user(username: str, token: str = Depends(oauth2_scheme)):
 
 username=alice&password=alice
 
-'''
\ No newline at end of file
+'''
+
+@router.post("/me/image/", response_model=UserOut, status_code=status.HTTP_200_OK)
+async def upload_profile_image(
+    request: Request,
+    file: UploadFile = File(...),
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)